src/main.rs

```diff
@@ -1,4 +1,4 @@
-use json_event_parser::{JsonEvent, JsonReader, JsonWriter};
+use json_event_parser::{FromReadJsonReader, JsonEvent, ToWriteJsonWriter};
```

Comment:
The build is failing, I think you forgot to include `json-event-parser` version bump in Cargo files.

```
error[E0432]: unresolved imports `json_event_parser::FromReadJsonReader`, `json_event_parser::ToWriteJsonWriter`
 --> src/main.rs:1:25
  |
1 | use json_event_parser::{FromReadJsonReader, JsonEvent, ToWriteJsonWriter};
  |                         ^^^^^^^^^^^^^^^^^^             ^^^^^^^^^^^^^^^^^ no `ToWriteJsonWriter` in the root
  |                         |
  |                         no `FromReadJsonReader` in the root

For more information about this error, try `rustc --explain E0432`.
error: could not compile `fil-deal-ingester` (bin "fil-deal-ingester") due to 1 previous error
```